babybuddy/middleware.py

```diff
@@ -88,3 +90,38 @@ class CustomRemoteUser(RemoteUserMiddleware):
     """
 
     header = getenv("PROXY_HEADER", "HTTP_REMOTE_USER")
+
+
+class HomeAssistant:
```

Comment:
I'd like to see more in code documentation here (and throughout the class) explaining what this is doing and _why_. I've spent a bit of time reviewing it and I *think* I understand based on various (weak) sources of information from HA forms and the like. Future me (and anyone else contributing code) would be thankful for more direct documentation here.